internal/saucecloud/zip/archive.go

```diff
@@ -0,0 +1,141 @@
+package zip
```

Comment:
Not bad, but...
1. Now we have two `zip` packages in the project.
2. Maybe better to move this back into `saucecloud` and keep it in the `archive.go` file.